plugins/ipam/host-local/host_local_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/containernetworking/cni/pkg/skel"
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/containernetworking/cni/pkg/types/020"
+	types020 "github.com/containernetworking/cni/pkg/types/020"
```

Comment:
Same, can you explain this change?

Reply:
Hmm i didn't notice this one. it might be vscode that made the change

Reply:
can you remove this then?